packages/shared-business/src/components/ChoicePicker.tsx

```diff
@@ -136,12 +136,16 @@ export const ChoicePicker = <T,>({
     }
 
     // auto scroll to selected value on mobile
-    const scrollContainer = containerRef.current;
+    const container = containerRef.current;
+    if (container == null) {
+      return;
+    }
+    const scrollContainer = container.element;
     const index = items.findIndex(item => value === item);
 
     if (index !== -1 && scrollContainer instanceof HTMLDivElement) {
```

Comment:
As `scrollContainer` is now a `HTMLElement | null`, perform a `!= null` instead to avoid an unnecessary cast.